soni/tests/acceptance/PageObjects/HomePage.js

```diff
@@ -0,0 +1,25 @@
+const { filesToDelete } = require('./../../../../testHelper/helper');
+class HomePage {
+    constructor() {
+        this.newFileSelector = "//button[@title='New file']"
+        this.textFieldSelector = "//input[@class='input input--block']"
+        this.createButtonSelector = "//button[@title='Create']"
+        this.contentFieldSelector = "//div[@class='ace_content']"
+        this.inputSelector =  ".ace_text-input"
+        this.saveButtonSelector = "//button[@aria-label='Save']"
+        this.closeButtonSelector = "//button[@aria-label='Close']"
+        // this.fileSelector = "//div[@aria-label='demo.txt']"
```

Comment:
if not needed remove this
```suggestion
        // this.fileSelector = "//div[@aria-label='demo.txt']"
```